metadata_backend/api/handlers.py

```diff
@@ -164,14 +164,16 @@ async def validate(self, req: Request) -> Response:
             raise web.HTTPBadRequest(reason=reason)
 
         except ParseError as error:
-            reason = f"Faulty XML file was given. Details: {error}"
-            body = json.dumps({"isValid": False, "reason": reason})
+            detail = f"Faulty XML file was given.<br/>ERROR: {error}"
```

Comment:
adding html-tags to responses is probably not a good idea, maybe just add real newline?